NASA-PDS/pds-registry-mgr-elastic
# Use schema from schemaLocation in a label file

## 🗒️ Summary
* Use schema from schemaLocation in a label file. (See https://github.com/NASA-PDS/pds-registry-mgr-elastic/issues/31)
* Added `list-dd` command to list loaded LDDs.

## ⚙️ Test Instructions
* NOTE: Use the latest Harvest and Registry-manager from `schema` branch.
* Delete old registry. Create new registry.
* Run Harvest as described in https://github.com/NASA-PDS/harvest/pull/72
* Run registry manager
* Check logs for loaded LDDs (mvn, particles)

## ♻️ Related Issues
https://github.com/NASA-PDS/harvest/pull/72
